CRM/Contact/BAO/Relationship.php

```diff
@@ -73,11 +76,66 @@ public static function create(&$params) {
       }
       CRM_Contact_BAO_Relationship::disableEnableRelationship($params['id'], $action);
     }
+
     self::addRecent($params, $relationship);
     return $relationship;
   }
 
   /**
+   * Create multiple relationships for one contact.
+   *
+   * The relationship details are the same for each relationship except the secondary contact
+   * id can be an array.
+   *
+   * @param array $params
+   *   Parameters for creating multiple relationships.
+   *   The parameters are the same as for relationship create function except that the non-primary
+   *   end of the relationship should be an array of one or more contact IDs.
+   * @param string $primaryContactLetter
+   *   a or b to denote the primary contact for this action. The secondary may be multiple contacts
+   *   and should be an array.
+   *
+   * @return array
+   * @throws \CRM_Core_Exception
+   */
+  public static function createMultiple($params, $primaryContactLetter) {
```

Comment:
$saved variable is set to 0 and never modified. This results in relationship edits being reported to user as 'Relationship created' in CRM_Contact_Form_Relationship::setMessage(). Broken webtest (WebTest_Member_InheritedMembershipTest.testInheritedMembership) revealed this issue.